.github/workflows/docker-tests.yml

```
@@ -0,0 +1,44 @@
+name: Continuous Build Docker
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

```suggestion
  pull_request:
    branches:
      - master
```

While we rarely have pull requests to other branches may as well stick to the branch